iidm/iidm-extensions/src/main/java/com/powsybl/iidm/network/extensions/TwoWindingsTransformerPhaseAngleClockAdderImpl.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.network.extensions;
+
+import com.powsybl.commons.extensions.AbstractExtensionAdder;
+import com.powsybl.iidm.network.TwoWindingsTransformer;
+
+/**
+ * @author Jérémy LABOUS <jlabous at silicom.fr>
+ */
+public class TwoWindingsTransformerPhaseAngleClockAdderImpl
+    extends AbstractExtensionAdder<TwoWindingsTransformer, TwoWindingsTransformerPhaseAngleClock>
+    implements TwoWindingsTransformerPhaseAngleClockAdder {
+
+    private int phaseAngleClock;
```

Comment:
Should we initialize this parameter to an invalid value, just to be sure the user set it manually?

Reply:
Yes, it should be set at -1